src/lib/mappers/invoice-account.js

```diff
@@ -31,7 +34,10 @@ const pojoToModelMapper = createMapper()
     'id',
     'accountNumber'
   )
-  .map('company').to('company', companyMapper.pojoToModel);
+  .map('company').to('company', companyMapper.pojoToModel)
+  .map('dateRange').to('dateRange', dateRangeMapper.pojoToModel)
+  .map('invoiceAccountAddresses').to('invoiceAccountAddresses',
```

Comment:
I've spotted another thing, but I'm not sure if this is the source of the problem:

The new invoice account workflow fails to render if the invoiceAccountAddresses are null/nonexistent. 

Stack below:
```
Debug: internal, implementation, error 
1|water-api    |     AssertionError [ERR_ASSERTION]: Address expected
1|water-api    |     at assertIsInstanceOf (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/models/validators.js:50:8)
1|water-api    |     at Role.set address [as address] (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/models/role.js:79:5)
1|water-api    |     at Role.fromHash (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/models/model.js:26:17)
1|water-api    |     at createModel (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/mappers/lib/helpers.js:14:16)
1|water-api    |     at crmToModel (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/mappers/role.js:28:31)
1|water-api    |     at Array.map (<anonymous>)
1|water-api    |     at Object.crmToModel (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/mappers/document.js:18:38)
1|water-api    |     at getDocument (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/services/documents-service.js:31:25)
1|water-api    |     at processTicksAndRejections (internal/process/task_queues.js:97:5)
1|water-api    |     at async module.exports.internals.Manager.execute (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/toolkit.js:45:28)
1|water-api    |     at async Object.internals.handler (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/handler.js:46:20)
1|water-api    |     at async exports.execute (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/handler.js:31:20)
1|water-api    |     at async Request._lifecycle (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/request.js:312:32)
1|water-api    |     at async Request._execute (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/request.js:221:9)
1|water-api    | 2020-10-15T11:40:13.193Z - error: 201015/114013.150, [error] message: Address expected, stack: AssertionError [ERR_ASSERTION]: Address expected
1|water-api    |     at assertIsInstanceOf (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/models/validators.js:50:8)
1|water-api    |     at Role.set address [as address] (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/models/role.js:79:5)
1|water-api    |     at Role.fromHash (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/models/model.js:26:17)
1|water-api    |     at createModel (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/mappers/lib/helpers.js:14:16)
1|water-api    |     at crmToModel (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/mappers/role.js:28:31)
1|water-api    |     at Array.map (<anonymous>)
1|water-api    |     at Object.crmToModel (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/mappers/document.js:18:38)
1|water-api    |     at getDocument (/Users/timtamimi/repos/water/water-abstraction-service/src/lib/services/documents-service.js:31:25)
1|water-api    |     at processTicksAndRejections (internal/process/task_queues.js:97:5)
1|water-api    |     at async module.exports.internals.Manager.execute (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/toolkit.js:45:28)
1|water-api    |     at async Object.internals.handler (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/handler.js:46:20)
1|water-api    |     at async exports.execute (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/handler.js:31:20)
1|water-api    |     at async Request._lifecycle (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/request.js:312:32)
1|water-api    |     at async Request._execute (/Users/timtamimi/repos/water/water-abstraction-service/node_modules/@hapi/hapi/lib/request.js:221:9)
1|water-api    | 2020-10-15T11:40:13.194Z - info: 201015/114013.150, [response] http://ip-192-168-68-106.eu-west-2.compute.internal:8001: get /water/1.0/licences/5cbfbfd4-2414-44bd-b819-f5f06cd43c94/valid-documents/2020-10-15 {} 500 (44ms)
```

Reply:
I think it's in a different mapper, but I have updated the related mappers to this issue to use the new library, so errors shouldn't be thrown if a data point is missing